superface/super.json

```diff
@@ -325,6 +325,20 @@
           "file": "../capabilities/language/analyze-plain-text-sentiment/maps/ibm-cloud-natural-language-understanding.suma"
         }
       }
+    },
+    "analytics/track-event": {
+      "file": "../capabilities/analytics/track-event/profile.supr",
+      "providers": {
+        "mixpanel": {
+          "file": "../capabilities/analytics/track-event/maps/mixpanel.suma"
+        },
+        "mock": {
+          "file": "../capabilities/analytics/track-event/maps/mock.suma"
+        }
+      },
+      "priority": [
```

Comment:
priority is not needed :)

Reply:
Added automatically by CLI. I didn't manually modified it.

Reply:
Yep, cli doesn't know we don't need it in station. I usually delete priority from added stuff in station super.json 🤷

Reply:
Removed